feature-libs/user/account/components/login/login.component.ts

```diff
@@ -35,4 +41,22 @@ export class LoginComponent implements OnInit {
       })
     );
   }
+
+  get greeting$() {
```

Comment:
I'm wary of using getter methods to communicate with the template because AFAIK, they will be called repeatedly to get the latest data. I think that we use it here so that we are always getting the up-to-date DOM and can react to changes. But I think this also has a large performance impact which will be sensitive to all applications (all users will likely have the login button in their storefronts).

Is it possible to use something like a `MutationObserver` or some other way to check for DOM updates that we can have in our `greeting$` pipe?